cdk/src/open-data-platform/lambda/write-demographic-data.handler.ts

```diff
@@ -75,36 +75,51 @@ async function connectToDb(secretArn: string): Promise<ConnectionPool> {
  */
 async function insertRows(
     db: Queryable, rows: DemographicsTableRow[]): Promise<any[]> {
-  const rowOptions:
-      BulkOperationOptions<'census_geo_id'|'total_population'|
-                           'black_percentage'|'white_percentage'> = {
-        database: db,
-        tableName: `demographics`,
-        columnTypes: {
-          census_geo_id: sql`VARCHAR`,
-          total_population: sql`REAL`,
-          black_percentage: sql`REAL`,
-          white_percentage: sql`REAL`,
-        },
-      };
+  // TODO(breuch): Replace with geom from new file.
```

Comment:
Can you mention in this comment what this raw value is?

Reply:
Done!